core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -152,7 +205,40 @@
                 ConfigDef.Importance.MEDIUM, MASTER_ELIGIBILITY_DOC)
         .defineOverride(METRICS_JMX_PREFIX_CONFIG, ConfigDef.Type.STRING,
                         METRICS_JMX_PREFIX_DEFAULT_OVERRIDE, ConfigDef.Importance.LOW,
-                        METRICS_JMX_PREFIX_DOC);
+                        METRICS_JMX_PREFIX_DOC)
+        .define(KAFKASTORE_SECURITY_PROTOCOL_CONFIG, ConfigDef.Type.STRING,
+                KAFKASTORE_SECURITY_PROTOCOL_PLAINTEXT, ConfigDef.Importance.HIGH,
+                KAFKASTORE_SECURITY_PROTOCOL_DOC)
```

Comment:
I made up my own priorities based on what felt good. I don't feel strongly about keeping them as is vs being consistent with Kafka. I'll make them consistent with Kafka unless I hear otherwise.